meson.build

```
@@ -462,15 +462,29 @@ endif
 ggml_dep = dummy_dep
 
 if get_option('enable-ggml')
+  message('preparing ggml')
   ggml_options = cmake.subproject_options()
+  ggml_root = meson.source_root() / 'subprojects' / 'ggml'
   if cxx_compiler_id == 'msvc'
     ggml_options.add_cmake_defines(common_win_subproject_cmake_defines)
   endif
   ggml_options.add_cmake_defines({'BUILD_SHARED_LIBS': host_machine.system() != 'windows'})
   ggml_options.add_cmake_defines({'GGML_BUILD_TESTS': false})
   ggml_options.add_cmake_defines({'GGML_BUILD_EXAMPLES': false})
+  ggml_options.add_cmake_defines({'CMAKE_INSTALL_LIBDIR': nntrainer_libdir / 'arm64-v8a'})
```

As I mentioned on #3118, this CMake define should only be added when the platform is Android.

```suggestion
if get_option('platform') == 'android'
  ggml_options.add_cmake_defines({'CMAKE_INSTALL_LIBDIR': nntrainer_libdir / 'arm64-v8a'})
endif
```